### ISSUE TITLE
I can't exit the intelligence item view with the action key as normal.

### ISSUE THREAD

**Issue by mzso** [haekb/nolf1-modernizer#31]
Hi!
It looks like with modernizer I can't do this. Which is annoying because the game doesn't get paused so I sometimes need to quickly exit it and I have to fumble for the escape key.

**Comment by haekb** [comment]
The game now pauses when you look at intelligence items in the game. This was one of the first things I added while working on Modernizer. I'll look into the action key problem though.

**Comment by mzso** [comment]
[**@haekb**](https://github.com/haekb) commented on [2020. máj. 5. 18:23 CEST](https://github.com/haekb/nolf1-modernizer/issues/31#issuecomment-624156191 "2020-05-05T16:23:26Z - Replied by Github Reply Comments"):
> The game now pauses when you look at intelligence items in the game. This was one of the first things I added while working on Modernizer. I'll look into the action key problem though.

I guess the camera sound continuing while viewing the item mislead me.

**Comment by fadster** [comment]
I agree. It would be nice to be able to exit the intelligence item view with the action button.

**Comment by Q3BFG10K** [comment]
I agree this would be nice to fix. Pressing 'Esc' is a little annoying.

**Comment by DanielGibson** [comment]
I've got a fix for this in a branch with a few improvements: https://github.com/DanielGibson/nolf1-modernizer/tree/improvements
It allows closing intelligence items with the right mouse button or the key on the keyboard that's bound to "activate".

**Comment by DanielGibson** [comment]
In case anyone wants to try my changes, download [MYDLLS.REZ.ZIP](https://github.com/haekb/nolf1-modernizer/files/11302839/MYDLLS.REZ.ZIP) and put the contained MYDLLS.REZ file into the Custom/ directory and make sure  to configure NOLF to load that .REZ (In NOLF launcher, Advanced -> Customize -> Select MYDLLS.REZ -> Add)

See https://github.com/DanielGibson/nolf1-modernizer/commits/improvements for the improvements I implemented

**Comment by DanielGibson** [comment]
From now on these changes (and an updated build) can be found in the following pull request: https://github.com/haekb/nolf1-modernizer/pull/55
